pkg/cli/debug/debug.go

```diff
@@ -452,6 +452,13 @@ func (o *DebugOptions) RunDebug() error {
 		ns = o.ToNamespace
 	}
 	pod.Name, pod.Namespace = fmt.Sprintf("%s-debug", generateapp.MakeSimpleName(infos[0].Name)), ns
+	// Must gather pods should not be preempted. Following the conventions mentioned at:
+	// https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#priority-classes
+	// setting priority class to system-cluster-critical
+	if strings.HasPrefix(pod.Namespace, "openshift-must-gather") {
```

Comment:
@stbenjam I'm quite hesitant on this fix, seems to much artificial, which actual script (I've noticed it's related with ovrit but couldn't find where) is actually trying to create the pods, that's rather unexpected b/c must-gather can't manage these pods and I'd discourage anyone trying to do so.

Reply:
It's ovirt?

These two collection scripts use oc debug directly:
  - https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_core_dumps#L11
  - https://github.com/openshift/must-gather/blob/master/collection-scripts/gather_multus_logs#L13

Reply:
@astoycos who added the first in https://github.com/openshift/must-gather/commit/e99c73060a94b592dfe29bffa9c4a900d2f616c2 and @nicklesimba who added the latter in https://github.com/openshift/must-gather/commit/4c55296fff7ad08b4fdb14a31940c828ec6970ce your approach is breaking our tests and is not the expected way to run pods. I see two path forward:
1. revert the changes
2. fix them, such that the test linked by @stbenjam in this PR doesn't fail